package.json

```diff
@@ -34,7 +34,7 @@
     "prettier": "^2.1.1",
     "rollup": "^2.26.9",
     "rollup-plugin-babel": "^4.3.3",
-    "rollup-plugin-postcss": "^3.1.8",
+    "rollup-plugin-postcss": "4.0.0",
```

Comment:
here is how to test this update: 

- create a semantic commit. this will do a pre-release of the pika-plugin-build-components-pkg
- go to the version tab on https://www.npmjs.com/package/@carforyou/pika-plugin-build-components and find the prerelease version
- go to components-pkg, install the prerelease
- do a semantic commit, wait for a prerelease of components-pkg
- install that components-pkg prerelease in listings
- test if styles in css modules from components are displayed properly

this is a lengthy process... but given it's a breaking change, I would probably go through the hassle.

from the changelog, it looks like they're now using postcss 8. I'm not sure if this would cause conflicts for us upstream: https://github.com/egoist/rollup-plugin-postcss/releases/tag/v4.0.0

Reply:
@masone 
Thank you for explaining the whole process 🙏 
I think I fixed the breaking change however in components-pkg (https://github.com/carforyou/carforyou-components-pkg/pull/374), I have this error on build : 
<img width="399" alt="npm error" src="https://user-images.githubusercontent.com/52455155/123958188-0916f080-d9ad-11eb-815e-c1412d1528a6.png">
I am not sure I should add this ?

Reply:
yes. it seems we already have babel/core installed for example in components, but the required version is higher than what we have installed. https://github.com/carforyou/carforyou-components-pkg/blob/master/package.json#L44

Reply:
@masone The CSS of the components with a CSS module is broken in listings. Does it make sense to invest some time trying to adjut to the breaking change knowing that the `cssnano` vulnerability is not fixed in the latest version of `rollup-plugin-postcss` ? At some point we might want to update `rollup-plugin-postcss` anyway I guess ?!